R/custom_disconnect_message.R

```diff
@@ -9,6 +9,7 @@
 #' @param publication_name The parent publication name
```

Comment:
line 5 in description should probably be `when dashboard disconnects` as it's not specific to RSConnect

Same with the return value in line 16/17

Reply:
Done